public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -260,16 +260,20 @@ export class DashboardModel {
     dispatch(onTimeRangeUpdated(timeRange));
   }
 
-  startRefresh() {
+  startRefresh(forceRefresh?: boolean) {
```

Comment:
We could avoid this parameter, as we have seen in https://github.com/percona-platform/grafana/pull/137/files

Reply:
Done 👍